src/System.Data.SqlClient/ref/project.json

```diff
@@ -5,7 +5,8 @@
     "System.IO": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
-    "System.Xml.ReaderWriter": "4.0.0"
+    "System.Xml.ReaderWriter": "4.0.0",
+	"System.Collections.NonGeneric": "4.0.0",
```

Comment:
If done as a separate item, would it mean the contract version would have to change again (e.g. `4.0.10.0` -> `4.0.11.0`) because the contract would be changing, or would it be OK to keep it `4.0.10.0` because it is just an intermittent change before an official "release"?